addons/web/static/tests/core/l10n/translation_tests.js

```diff
@@ -1,11 +1,15 @@
 /** @odoo-module **/
 
+import { browser } from "@web/core/browser/browser";
 import { translatedTerms, _lt } from "@web/core/l10n/translation";
+import { localizationService } from "@web/core/l10n/localization_service";
 import { registry } from "@web/core/registry";
 import { patch, unpatch } from "@web/core/utils/patch";
+import { session } from "@web/session";
 import { makeTestEnv } from "../../helpers/mock_env";
 import { makeFakeLocalizationService } from "../../helpers/mock_services";
-import { getFixture } from "../../helpers/utils";
+import { getFixture, patchWithCleanup } from "../../helpers/utils";
+import { registerCleanup } from "../../helpers/cleanup";
```

Comment:
Didn't we want to get rid of relative imports? Or is it irrelevant in test suites?